app/lib/supergroups_api/supergroup_collection.rb

```diff
@@ -0,0 +1,29 @@
+module SupergroupsApi
+  class SupergroupCollection
+    SUPERGROUP_TYPES = %w(
```

Comment:
Is it worth reading the supergroups from here: https://github.com/alphagov/govuk_document_types/blob/master/data/supergroups.yml in case they change?

Reply:
@leenagupte I thought about it, but it [says here](https://github.com/alphagov/govuk_document_types#govuk-document-types):

> This gem is only to be used in publishing-api and rummager. **Don't use it in your project.**

Reply:
I think @sihugh was thinking about ways to make the supergroups my dynamic. I'm not sure how far he got though.

Reply:
That would be great. The stern warning on govuk_document_types scared me off!